client/gutenberg/editor/controller.js

```diff
@@ -42,6 +48,36 @@ function registerDataPlugins( userId ) {
 	use( plugins.controls );
 }
 
+export const jetpackBlocki18n = ( context, next ) => {
```

Comment:
Well, I think I'll pass because:

* We need access to the state, in order to fetch the user's current locale.
* For the suggested approach, we need to pause the flow and continue after translations are loaded.

Reply:
We could pass the state (or part of it) to `initGutenberg` from controller, but I agree that we shouldn't do it here because of your second point.